src_kt/java/arcs/storage/BUILD

```diff
@@ -36,6 +37,7 @@ kt_jvm_and_js_library(
     name = "storage_key",
     srcs = [
         "StorageKey.kt",
+        "StorageKeyParser.kt",
     ],
```

Comment:
Optional: You could define a `STORAGE_KEY_SRCS` const for these, and then you wouldn't need to be so careful to duplicated the same list of srcs here and in `excludes` above.

Reply:
Thanks, nice tip!